tsdb/engine/tsm1/encoding.go

```diff
@@ -220,11 +239,13 @@ func encodeFloatBlock(buf []byte, values []Value) ([]byte, error) {
 	// for timestamps and values.
 
 	// Encode values using Gorilla float compression
-	venc := NewFloatEncoder()
+	venc := getFloatEncoder()
+	defer putFloatEncoder(venc)
```

Comment:
I don't think I need it in the error block if it's done right after the `Bytes` calls.